pilot/pkg/model/push_context.go

```diff
@@ -785,7 +785,6 @@ func (ps *PushContext) GatewayServices(proxy *Proxy) []*Service {
 }
 
 // add services from MeshConfig.ExtensionProviders
-// TODO: include cluster from EnvoyFilter such as global ratelimit [demo](https://istio.io/latest/docs/tasks/policy-enforcement/rate-limit/#global-rate-limit)
```

Comment:
It does when _directly_ adding the cluster via EnvoyFilter ,but not if you just point to a cluster created by a Service I think.

Reply:
do you have an example for that?

Reply:
Sure, in the linked rate limit  example just replace `cluster_name: rate_limit_cluster` with something like `cluster_name: outbound|80||ratelimit.ns.svc.cluster.local`. IMO this is actually a better pattern because it allows using standard Istio functionality like mtls, etc easily

Reply:
> Sure, in the linked rate limit example just replace `cluster_name: rate_limit_cluster` with something like `cluster_name: outbound|80||ratelimit.ns.svc.cluster.local`. IMO this is actually a better pattern because it allows using standard Istio functionality like mtls, etc easily

this remind me about something I forgot in https://github.com/istio/istio/issues/35398#issuecomment-1184664572

I will send a PR to fix the doc, let's keep this TODO.